coordinator/tests/services/translation/request/dispense/release.spec.ts

```diff
@@ -0,0 +1,30 @@
+import * as fhir from "../../../../../src/models/fhir"
+import {translateReleaseRequest} from "../../../../../src/services/translation/request/dispense/release"
+
+describe("translateReleaseRequest", () => {
+  const parameters = new fhir.Parameters([{
+    "name": "owner",
+    "valueIdentifier": {
+      "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+      "value": "VNE51"
+    }
+  }])
+  const translatedRelease = translateReleaseRequest(parameters)
+
+  test("translated release contains agentPersonPerson and representedOrganization", () => {
+    const author = translatedRelease.NominatedPrescriptionReleaseRequest.author
+    expect(author).not.toBeUndefined()
```

Comment:
I think this assertion will still allow null. Maybe use `toBeTruthy()` instead?

Reply:
Huh. Makes sense when you put it like that `expect(null).not.toBeUndefined()`

Reply:
Ill fix it in Conner's absence